### DISCUSSION TITLE
User or Device 'Last Seen' /'Checkin'

### DISCUSSION THREAD

**Discussion by SCUZNUTS** [immich-app/immich#5742] [General]
It would be good to know if my partner or family member phones etc are still checking in and uploading on the regular.

**Comment by jtagcat** [comment]
adding search dedupe keywords: backups backing up regularly admin battery optimization backup status

I'm planning to replace Syncthing with Immich for the family device fleet. Feature parity would be:
- [ ] Showing count/**date** of unsynced items (per device, not user!)
![image](https://github.com/user-attachments/assets/88b54877-f863-4647-ab02-e87817fd32d0)
- [x] Showing last seen (already exists)
![image](https://github.com/user-attachments/assets/355833bc-3765-4d5d-bb26-ba4af3ef2c8d)
- [ ] Exposing the overview in a single place (not collecting passwords and logging in to each one). Could be a dashboard, could be Prometheus.

Some devices check in once every week or two, some do it every day. I need to know if there is intervention required, and for better over-the-phone instructions, what caused it (is the app dead and just needs to be opened, is WiFi always off while charging (some always stay on mobile data).
